cmd/schemagen/schemagen.go

```diff
@@ -19,19 +20,19 @@ import (
 )
 
 var (
-	cmd          = flag.NewFlagSet(os.Args[0], flag.ExitOnError)
-	flagCluster  = cmd.String("cluster", "127.0.0.1", "a comma-separated list of host:port tuples")
-	flagKeyspace = cmd.String("keyspace", "", "keyspace to inspect")
-	flagPkgname  = cmd.String("pkgname", "models", "the name you wish to assign to your generated package")
-	flagOutput   = cmd.String("output", "models", "the name of the folder to output to")
-	flagUser     = cmd.String("user", "", "user for password authentication")
-	flagPassword = cmd.String("password", "", "password for password authentication")
+	cmd               = flag.NewFlagSet(os.Args[0], flag.ExitOnError)
+	flagCluster       = cmd.String("cluster", "127.0.0.1", "a comma-separated list of host:port tuples")
+	flagKeyspace      = cmd.String("keyspace", "", "keyspace to inspect")
+	flagPkgname       = cmd.String("pkgname", "models", "the name you wish to assign to your generated package")
+	flagOutput        = cmd.String("output", "models", "the name of the folder to output to")
+	flagUser          = cmd.String("user", "", "user for password authentication")
+	flagPassword      = cmd.String("password", "", "password for password authentication")
+	flagIgnore        = cmd.String("ignore", "", "a comma-separated list of table, view or index names to ignore")
+	flagIgnoreIndexes = cmd.Bool("ignore-indexes", false, "a comma-separated list of indexes to ignore")
 )
 
-var (
-	//go:embed keyspace.tmpl
-	keyspaceTmpl string
-)
+//go:embed keyspace.tmpl
+var keyspaceTmpl string
```

Comment:
This refactor has nothing to do with actual PR. Not sure if we should mix adding new features and refactoring

Reply:
Agree, that was cause by my IDE setup, I'll revert it.